README.md

```diff
@@ -33,6 +33,16 @@ ssh -NL 64444:localhost:5432 -o "ExitOnForwardFailure yes" -o ServerAliveInterva
 
 <br />
 
+#### Using Autossh
```

Comment:
I think this is a great recommendation but I have a question. 

I originally made this lib to spin up an ssh connection only while my service is running, otherwise there is no open connection. My original problem with running autossh on my servers was that it was so separate from my application code and didn't have any insight in its status without going into my machine and looking at logs. 

Would this be able to still spin up an ssh connection and kill it when my service dies? Also, if the "shell autossh" dies and restarts, will our node service still have a reference to the process so that it can kill it when my service dies?

Reply:
If you kill autossh, ssh dies. If ssh dies through a deliberate kill autossh dies as well, otherwise it will reconnect.